godot-core/src/builtin/plane.rs

```diff
@@ -105,6 +107,90 @@ impl Plane {
         }
     }
 
+    /// Returns the shortest distance from the `self` `Plane` to the provided `Vector3`. The distance 
+    /// will be positive if the `Vector3` is above the `Plane`, and will be negative if the `Vector3` 
+    /// is below the `Plane`.
+    ///
+    /// _Godot equivalent: `Plane.distance_to(Vector3 point)`_
+    #[inline]
+    pub fn distance_to(&self, point: Vector3) -> real {
+        self.normal.dot(point) - self.d
+    }
+
+    /// Returns the center of the `self` `Plane` in `Vector3` form. 
+    ///
+    /// _Godot equivalent: `Plane.get_center()`_
+    #[inline]
+    pub fn get_center(&self) -> Vector3 {
+        self.normal * self.d
+    }
+
+    /// Returns `true` if the specified `Vector3` is inside the `self` `Plane`, will return otherwise
+    /// if not. Tolerance of the function unless specified will be using the default value of 1e-05.
+    ///
+    /// _Godot equivalent: `Plane.has_point(Vector3 point, float tolerance=1e-05)`_
+    #[inline]
+    pub fn has_point(&self, point: Vector3, tolerance: Option<real>) -> bool {
```

Comment:
Consistent with other geometrics, this should be "contains_point", with an alias to find it under "has_point" in API docs:
```suggestion
    #[doc(alias = "has_point")]
    pub fn contains_point(&self, point: Vector3, tolerance: Option<real>) -> bool {
```

Reply:
And from looking at the others, this is the same case as well for `get_center()` -> `center()`?

Reply:
Yes, getters in Rust generally don't have the `get_` prefix (I know we're not 100% consistent yet, but it's getting better...)
For omitting getter prefix, you don't need a `#[doc(alias)]` though. It's really for different naming.